news.model.js

```diff
@@ -1,29 +1,42 @@
-const db = require('./db/connection')
-const response = require('./app');
-const comments = require('./db/data/test-data/comments');
-const articles = require('./db/data/test-data/articles');
+const db = require("./db/connection");
+const response = require("./app");
+const comments = require("./db/data/test-data/comments");
+const articles = require("./db/data/test-data/articles");
 
```

Comment:
`response`, `comments`, `articles`, why are these here? They're not being used.

Reply:
Is response not being used? My bad with the other 2, forgot to remove when figuring stuff out but not being used so will remove on my next branch c: